@hts-forking/test/getHtsStorageAt.test.js

```diff
@@ -38,7 +38,10 @@ describe('::getHtsStorageAt', function () {
 
     /** @type {IMirrorNodeClient} */
     const baseMirrorNodeClient = {
-        getTokenById(_tokenId) {
+        async getTokenById(_tokenId) {
+            throw Error('Not implemented');
```

Comment:
Q: Why are these tests not implemented? Is there a plan to?

Reply:
Not sure why they are not implemented yet, but it looks like it’s out of scope for this PR

Reply:
> Q: Why are these tests not implemented? Is there a plan to?

These are meant to be `Not implemented`. This `baseMirrorNodeClient` acts as a base object to be used in subsequent tests where you override what it's needed for said test. This is to ensure no unwanted calls to the Mirror Node are made.

Reply:
So it's more of a stub,

 I suggest we rename it to `mirrorNodeClientStub` or `mockMirrorNodeClient` to make things clearer.